client/src/resources/lang/en_us.json

```diff
@@ -434,6 +434,9 @@
     "cult:var.0": "Cultist",
     "cult:var.1": "Cultists",
     "neutral": "Neutral",
+    "fiends": "Fiends",
+    "fiends:var.0": "Fiend",
+    "fiends:var.1": "Neutral Killing",

```

Comment:
Remove this. You don't have "sheriff" as an alt for detective or "executioner" as an alt for hater